webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,21 @@ function BugForm() {
     }, this));
   };
 
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      var savedValue = this.value;
+      this.value = "";
+      this.baseScrollHeight = this.scrollHeight;
+      this.value = savedValue;
```

Comment:
I was worried we might lose the comment data but it turns out the `savedValue` variable is not even needed.